lib/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -122,10 +123,11 @@ public function update_item( $request ) {
 			update_post_meta( $data['id'], '_wp_attachment_image_alt', $request['alt_text'] );
 		}
 
-		$response = $this->get_item( array(
-			'id'      => $data['id'],
-			'context' => 'edit',
-		));
+		$attachment_location = rest_url( '/wp/v2/' . $this->get_post_type_base( get_post_type( $data['id'] ) ) . '/' . $data['id'] );
```

Comment:
THere's now harm, it's just added complexity, and I'd rather that creating a `WP_REST_Request` manually wasn't a PITA, so the less stuff the better IMO. However, happy to be convinced otherwise, just based of simpler-is-better, not needing to "know" the URL of the resource is good.